src/index.ts

```diff
@@ -95,7 +97,11 @@ export default class SwupPreloadPlugin extends Plugin {
 		if (typeof preloadVisibleLinks === 'object') {
 			this.options.preloadVisibleLinks = {
 				...this.options.preloadVisibleLinks,
-				...preloadVisibleLinks
+				...preloadVisibleLinks,
+				enabled:
+					typeof preloadVisibleLinks.enabled === 'boolean'
```

Comment:
As we're already checking above that it's an object, this should be enough to enable it, but allow disabling it inside the object:

```js
this.options.preloadVisibleLinks = {
				...this.options.preloadVisibleLinks,
				enabled: true,
				...preloadVisibleLinks
}

Reply:
ah right!